internal/integration-tests/equipment/integration_test.go

```diff
@@ -504,7 +504,7 @@ func TestIntegration_BlockEquipment(t *testing.T) {
 
 	ctx := context.Background()
 	client := utils.SetupClient()
-	startDate, endDate := strfmt.DateTime(time.Now()), strfmt.DateTime(time.Now().AddDate(0, 0, 10))
+	startDate, endDate := strfmt.DateTime(time.Now().AddDate(0, 0, 1)), strfmt.DateTime(time.Now().AddDate(0, 0, 10))
```

Comment:
I don't see any new integration tests here for the new case. Could you please create one?